README.md

```diff
@@ -101,7 +101,7 @@ Model galleries
 - https://github.com/go-skynet/model-gallery
   
 UI / Management Programs
-- [LocalAI Manager](https://io.midori-ai.xyz/howtos/easy-model-installer/)
+- [Midori AI Subsystem Manager](https://io.midori-ai.xyz/subsystem/manager/)
```

Comment:
this list here is specifically for projects that directly are based off from LocalAI - this now sounds more should go in the integration section instead of the docs

Reply:
@mudler how would I do that?